app/uk/gov/hmrc/soletraderidentificationfrontend/connectors/CreateNinoIVJourneyConnector.scala

```diff
@@ -64,12 +65,11 @@ class CreateNinoIVJourneyConnector @Inject() (http: HttpClient, appConfig: AppCo
         )
       )
 
-    http.POST[JsObject, NinoIVJourneyCreationResponse](appConfig.createNinoIVJourneyUrl, jsonBody)(
-      implicitly[Writes[JsObject]],
-      NinoIVHttpReads,
-      hc,
-      ec
-    )
+    httpClient
+      .post(url = url"${appConfig.createNinoIVJourneyUrl}")(hc)
```

Comment:
hc is implicit so it shouldn't need to be passed in.

Reply:
Looking at it the previous code they have been passed in due to the "implicitly[Writes[JsObject]]" being passed in.